CHANGELOG.md

```diff
@@ -1,3 +1,79 @@
+## Release 0.16.1
+
+This is a bugfix release. In addition, the security of Scope probes can be
+hardened through the new flag `--probe.no-controls` with which controls will be
+disabled. Without controls, Scope users won't be able to open terminals,
+start/stop containers, show logs ...
```

Comment:
"In addition, the security of the Scope probe can be hardened by disabling controls with the new `--probe.no-controls` flag. Without controls, Scope users won't be able to open terminals, start/stop containers, view logs."